### DISCUSSION TITLE
[SOLVED] version 2.1 doesn't start when using odoh

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#1820] [Potential issues]
Thanks for the new version.

It works fine with my normal configuration. However I don't get the proxy to start when trying to test odoh (error message just says it wasn't started).

Is there something special to do other than specifying a route, uncommenting the odoh sources, setting odoh_servers = true and restarting the proxy?

**Comment by kokial** [comment]
Hi !
Have you added these brand new sources ?

[sources.'odoh-relays']
urls = ['https://raw.githubusercontent.com/DNSCrypt/dnscrypt-resolvers/master/v3/odoh-relays.md', 'https://download.dnscrypt.info/resolvers-list/v3/odoh-relays.md']
minisign_key = 'RWQf6LRCGA9i53mlYecO4IzT51TGPpvWucNSCh1CBM0QTaLn73Y7GFO3'
cache_file = 'odoh-relays.md'

[sources.'odoh-servers']
urls = ['https://raw.githubusercontent.com/DNSCrypt/dnscrypt-resolvers/master/v3/odoh-servers.md', 'https://download.dnscrypt.info/resolvers-list/v3/odoh-servers.md']
minisign_key = 'RWQf6LRCGA9i53mlYecO4IzT51TGPpvWucNSCh1CBM0QTaLn73Y7GFO3'
cache_file = 'odoh-servers.md'

**Comment by ghost** [comment]
Works fine now. Maybe I had a typo in the toml. I just redid it from scratch with the same settings.
